### PR TITLE
Supported reporting through dd-agent.

### PR DESCRIPTION
**Pull request by davewgithub**
This commit adds support for datadog reporting through datadog agent.

### CODE REVIEW

File: akka-app/src/ooyala.common.akka/metrics/DatadogConfigParser.scala [hunk 1]
```diff
@@ -31,12 +33,13 @@ object DatadogConfigParser {
...
    *
-   * Parses the reporting host name, api key, tags, and duration from the Datadog configuration
-   * section. If the host name is not set, sets it to the local host name.
+   * Parses the reporting host name, datadog agent port, api key, tags, and duration from the
+   * Datadog configuration section. If the host name is not set, sets it to the local host name.
    *
    *  Example config setting in spark.jobserver.metrics.datadog
    *  spark.jobserver.metrics.datadog {
    *    hostname = example
+   *    agentport = 9999
```
Comment (myxjtu) [inline akka-app/src/ooyala.common.akka/metrics/DatadogConfigParser.scala]: Minor suggestion: change the port number "9999" to a default port value (8125?)

File: project/Dependencies.scala [hunk 2]
```diff
@@ -57,7 +57,7 @@ object Dependencies {
   lazy val apiDeps = sparkDeps :+ typeSafeConfigDeps
   lazy val monitoringDeps = Seq(
     "com.codahale.metrics" % "metrics-core" % "3.0.1",
-    "org.coursera" % "metrics-datadog" % "1.0.1",
+    "org.coursera" % "metrics-datadog" % "1.0.2",
```
Comment (myxjtu) [inline project/Dependencies.scala]: could u add comments why upgrading metrics-datadog?

File: akka-app/src/ooyala.common.akka/metrics/DatadogConfigParser.scala (hunk 1 above)
Comment (davewgithub) [inline akka-app/src/ooyala.common.akka/metrics/DatadogConfigParser.scala]: Updated in fa5d6ae

File: project/Dependencies.scala (hunk 2 above)
Comment (davewgithub) [inline project/Dependencies.scala]: Added comments in fa5d6ae

**Pull request merged**